src/consumer.ts

```diff
@@ -134,6 +139,31 @@ export class Consumer extends TypedEventEmitter {
     return !this.stopped;
   }
 
+  /**
+   * Updates the provided option to the provided value.
+   * @param option The option that you want to update
+   * @param value The value to set the option to
+   */
+  public updateOption(
+    option: UpdatableOptions,
+    value: ConsumerOptions[UpdatableOptions]
+  ) {
+    switch (option) {
+      case 'visibilityTimeout':
+        if (typeof value === 'number') {
```

Comment:
We probably also need to test that this value is not less than `heartbeatInterval` if that value has been defined.